### PR TITLE
[Server] Refactoring of Function Comments

### PR DESCRIPTION
**Pull request by dipankardas011**
server/handlers/content_modifier

Signed-off-by: Dipankar Das <dipankardas0115@gmail.com>

**Description**
Fixing the function comment

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by acald-creator — APPROVED** [review]

File: server/handlers/content_modifier.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func NewContentModifier(token string,
 	}
 }
 
-// TODO: Similar mechanisms for filters and applications
+// AddMetadataForPatterns TODO: Similar mechanisms for filters and applications
 // Takes in response bytes, and add metadata to it based on some checks
```
Comment (Abhishek-kumar09) [inline server/handlers/content_modifier.go:34]: It should be `AddMetadataForPatterns takes in response bytes, and add metadata to it based on some checks`

Comment (dipankardas011) [reply server/handlers/content_modifier.go:34]: Okay 👍🏼

**Comment by leecalcote** [comment]
Thank you, @dipankardas011 👍

**Comment by dipankardas011** [comment]
> Thank you, @dipankardas011 +1

I will stay in this project for long 🤞🏼

**Comment by leecalcote** [comment]
Awesome. Good work, @dipankardas011 👏

**Pull request merged**